### PR TITLE
⚡️(api) add user password reset script

### PR DESCRIPTION
**Pull request by jmaupetit**
## Purpose

Because people forget...

## Proposal

This scripts reset user password and send the new password in a Vault Sent. Vault infornations are sent by email.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -285,7 +285,7 @@ jupytext--to-ipynb: ## convert remote md files into ipynb
 
 reset-db: ## Reset the PostgreSQL database
 	$(COMPOSE) stop
-	$(COMPOSE) down postgresql metabase
+	# $(COMPOSE) down postgresql metabase
```
Comment (jmaupetit) [inline Makefile:288]: Should be removed in a separated commit. Too many issues using this.

Comment (ssorin) [inline Makefile:288]: dead code or forgot to uncomment?

File: scripts/create-api-users.sh [hunk 2]
```diff
@@ -86,138 +78,7 @@ function api_user() {
...
-      --arg firstname "${firstname}" \
-      --arg send_password "${send_password}" \
-      -f <(echo "${tpl}")
-  )
-
-  brevo_email "${payload}"
-}
-
-# Create users and groups
+# Create users
```
Comment (ssorin) [inline scripts/create-api-users.sh:81]: Create users -> Create groups

File: scripts/create-api-users.sh [hunk 3]
```diff
@@ -233,10 +94,11 @@ function create_groups() {
     name=$(echo "${name}" | xargs)
     ou=$(echo "${ou}" | xargs)
 
-    api_group "${name}" "${ou}"
+    create_group "${name}" "${ou}"
   done < <(tail -n +2 "${GROUPS_CSV_FILENAME}")
 }
 
+# Create groups
```
Comment (ssorin) [inline scripts/create-api-users.sh:101]: # Create groups -> Create users

File: scripts/reset-password.sh [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+# export BW_CLIENTID
+# export BW_CLIENTSECRET
+# export BW_PASSWORD
+#
+# Brevo
+#
+# export BREVO_API_KEY
+#
+
+set -eo pipefail
```
Comment (ssorin) [inline scripts/reset-password.sh:26]:
```suggestion
set -euo pipefail
```

File: scripts/_common.sh [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+  echo "${send_url}"
+}
+
+# Send a transactional email using Brevo
+function brevo_email() {
+
+  local payload="${1}"
+
+  curl --request POST \
+    --url https://api.brevo.com/v3/smtp/email \
```
Comment (ssorin) [inline scripts/_common.sh:68]: could we use a constant for 'https://api.brevo.com/v3/smtp/email' ?

File: scripts/reset-password.sh (hunk 4 above)
Comment (jmaupetit) [reply scripts/reset-password.sh:26]: I removed it as I am checking command usage. Using `-u` option prevent raising an explicit error message.

File: Makefile (hunk 1 above)
Comment (jmaupetit) [reply Makefile:288]: This is something I need to fix separately.

**Review by ssorin — APPROVED** [review]

**Pull request merged**